src/SaveObject.ts

```diff
@@ -38,20 +38,27 @@ import { Terminal } from "./Terminal";
 import { getRecordValues } from "./Types/Record";
 import { ExportMaterial } from "./Corporation/Actions";
 import { getGoSave, loadGo } from "./Go/SaveLoad";
+import { SaveData } from "./types";
+import { decodeSaveData, encodeJsonSaveString } from "./utils/saveDataUtils";
+import { isBinaryFormat } from "../electron/saveDataBinaryFormat";
 
 /* SaveObject.js
  *  Defines the object used to save/load games
  */
 
-export interface SaveData {
+/**
+ * This interface is only used for transferring game data to electron-related code, so I rename it from SaveData to
+ * ElectronGameData.
```

Comment:
```suggestion
 * This interface is only for transferring game data to electron-related code.
```

Code comments should reflect the (current) state of the code, you can use PR comments to discuss/justify changes being made in a particular change.